pkgs/desktops/gnome-3/extensions/text-translator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, gnome3, pkgs, substituteAll, translate-shell}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-text-translator";
+  version = "unstable-2020-02-25";
+
+  src = fetchFromGitHub {
+    owner = "gufoe";
+    repo = "text-translator";
+    rev = "991daadff538bd7acd15cf60415704e0215f2bf6";
+    sha256 = "098a967p1gr8g20nz0fdnlmpc6k40fh0ijqcdhp7ckfqr39imkhr";
+  };
+
+  uuid = "text_translator@awamper.gmail.com";
+
+  patches = [
+    (substituteAll {
+      src = ./fix-trans.patch;
+      trans = "${translate-shell}/bin/trans";
+    })
+  ];
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/${uuid}
+    cp -r . $out/share/gnome-shell/extensions/${uuid}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A text translator extension for GNOME Shell";
+    license = licenses.unlicense;
```

Comment:
Where do you see the license in the package?

Reply:
I did not find any license on the project, that's why I added this `licenses.unlicense`, am I wrong?

Reply:
[Unlicense](https://unlicense.org/) is a specific license. When no license is declared, it is considered All rights reserved by law and should use `licenses.unfree`.

Reply:
Upstream probably just forgot to add a license, they should be poked about it.

Reply:
I didn't know about that, thanks @jtojnar !

Reply:
Done @veprbl